server/adaptors/integrations/types.ts

```diff
@@ -124,6 +124,7 @@ interface AssetReference {
   assetId: string;
   isDefaultAsset: boolean;
   description: string;
+  status?: string;
```

Comment:
Can the status be from an enum?

Reply:
I have a few things to fix with status so I made #1545, will work on it soon